vcs/util/path.go

```diff
@@ -0,0 +1,12 @@
+package util
+
+import "strings"
+
+// Rel strips the leading "/" prefix from the path string, effectively turning
+// an absolute path into one relative to the root directory.
```

Comment:
Minor, the "/" -> "." special case isn't documented here.

Also, I think this package would be a good candidate to place in `vcs/internal/util` instead of `vcs/util`. It's an internal utility that is used by multiple vcs implementations, but it's not really a part of the API of this repository and we'd want to be free to change/remove it without worrying about external users.